docs/getting-started/supported-languages-and-tools.md

```diff
@@ -118,7 +118,7 @@ The table below lists all languages and frameworks that Codacy supports and the
     </tr>
     <tr>
       <td>Dart</td>
-      <td><a href="https://github.com/dart-lang/sdk/tree/main/pkg/analyzer_cli">dartanalyzer</a></td>
+      <td><a href="https://github.com/dart-lang/sdk/tree/main/pkg/analyzer_cli">dartanalyzer</a><a href="dart-limitations"><sup>5</sup></a></td>
```

Comment:
- There's a missing `#` on the fragment name (that's what caused the `mkdocs` CI step to fail)
- Let's increment the footnotes by order of appearance - this is a big table and it's already becoming hard to find where each footnote is. :slightly_smiling_face:. So, this would be footnote 3, and the current 3 and 4 need to become 4 and 5:

```suggestion
      <td><a href="https://github.com/dart-lang/sdk/tree/main/pkg/analyzer_cli">dartanalyzer</a><a href="#dart-limitations"><sup>3</sup></a></td>
```